docs/src/tutorial/genomic-surveillance.rst

```diff
@@ -0,0 +1,158 @@
+Run the workflow using a genomic surveillance configuration
+===========================================================
+
+In this tutorial, you will replace the custom contextual data with meaningful focal data and use a configuration commonly found in genomic surveillance applications.
+
+.. contents:: Table of Contents
+   :local:
+
+Prerequisites
+-------------
+
+1. :doc:`custom-data`. These instructions will introduce the workflow configuration that is modified in this tutorial.
+
+Setup
+-----
+
+If you are not already there, change directory to the ``ncov`` directory:
+
+   .. code:: text
+
+      cd ncov
+
+Curate data from GISAID
+-----------------------
+
+.. warning::
+
+   These steps require a GISAID account. `Register <https://www.gisaid.org/registration/register/>`__ if you do not have an account yet. However, registration may take a few days. Follow `alternative data preparation methods <../guides/data-prep.html>`__ if you wish to continue this tutorial in the meantime.
+
+We will download a custom focal set from GISAID's EpiCoV database.
+
+1. Navigate to `GISAID <https://www.gisaid.org/>`__, **Login**, and go to **EpiCoV** > **Search**.
+
+   .. image:: ../images/gisaid-epicov-search.png
+      :width: 400
+      :alt: GISAID EpiCoV Search
+
+2. Select a custom set of records. As an example, this guide will look for all records that pass the following criteria:
+
+   1. From Idaho
+   2. Collected between ``2022-02-15`` and ``2022-03-15``
+   3. Has a complete genome
+   4. Has an exact collection date
```

Comment:
We need a step 5 here that tells users to click the checkbox in the top left to select all records (across all search result pages).

Reply:
Done: 460fc340